config/esp32/components/chip/CMakeLists.txt

```diff
@@ -29,7 +29,11 @@ if(NOT CHIP_ROOT)
     get_filename_component(CHIP_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../../../.. REALPATH)
 endif()
 
-set(CHIP_REQURIE_COMPONENTS freertos lwip bt mdns mbedtls fatfs)
+set(CHIP_REQURIE_COMPONENTS freertos lwip bt nvs_flash mdns mbedtls fatfs)
```

Comment:
What has changed that previously `nvs_flash` was not required and now it is required?

Reply:
Removed